wrappers/javascript/anoncreds-nodejs/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "types": ["node", "jest"]
+    "types": ["node", "jest"],
+    "skipLibCheck": true
```

Comment:
Yeah I wanted to add a comment but its json :). 

This was done because, unsure why, there was a collision from `@types/ref-napi` between the packages own `node_modules/@types/ref-napi` and the workspaces `node_modules/@types/ref-napi`. 

Very unsure about resolving this issue. It did not occur before as the validation script passed.

If this is a familiar issue for you we can def revert it, I just was not sure how.

Reply:
How can I reproduce the error?

Reply:
I think:

```console
git clone https://github.com/blu3beri/anoncreds-rs -b nodejs-release-ready
cd anoncreds-rs/wrappers/javascript
yarn
yarn check-types
```

Reply:
setting `@types/ref-napi` in package.json for the nodejs wrapper to `^3.0.6` seems to have resolved the issue